test/extended/util/docker.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import (
+	dockerClient "github.com/fsouza/go-dockerclient"
+	tutil "github.com/openshift/origin/test/util"
+)
+
+func CorruptImage(corruptee, corruptor string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.TagImageOptions{
+		Repo:  corruptee,
+		Tag:   "latest",
+		Force: true,
+	}
+	terr := client.TagImage(corruptor, opts)
+	return terr
+}
+
+func ResetImage(name string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.PullImageOptions{
+		Repository: name,
+		Tag:        "latest",
+	}
+	perr := client.PullImage(opts, dockerClient.AuthConfiguration{})
+	return perr
+}
+
+func CompareImages(comp1, comp2 string, returnID1, returnID2 *string) error {
```

Comment:
I think using the multiple return values is the way to go here.

a:= ""
b:=""
someMethod(param1,&a,&b)

seems ugly/inconvenient compared to:
a,b:=someMethod(param1)

and generally using parameters to return values feels like the wrong technique since it's harder for someone using the method to understand the result is coming back in the reference they passed in, rather than as a return value.

it's definitely not a "huge" advantage, but it does seem like the more straightforward approach.